CONTRIBUTING.md

```diff
@@ -2,33 +2,22 @@
 
 Contributions are highly welcome, however, except for very small changes, kindly file an issue and let's have a discussion before you open a pull request.
 
-## Building The Project
+## Building the Project
```

Comment:
Could you move this file under `Documentation` and link to readme?
@tonerdo this lead to simple conflics in https://github.com/tonerdo/coverlet/pull/422/files#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
Github only recognizes this specially named file if it's in the root, I think.

Reply:
> Github only recognizes this specially named file if it's in the root, I think.

Mmm yes it's a shame https://help.github.com/en/articles/setting-guidelines-for-repository-contributors#adding-a-contributing-file root or doc